ui-lib/src/androidTest/java/co/electriccoin/zcash/ui/screen/backup/integration/BackupActivityTest.kt

```diff
@@ -34,18 +34,18 @@ class BackupActivityTest : UiTestPrerequisites() {
         val testSetup = newTestSetup()
         testSetup.setDefaultContent()
 
-        assertEquals(BackupStage.EducationOverview, testSetup.getStage())
+        assertEquals(BackupStage.EducationOverview.order, testSetup.getStage().order)
 
         composeTestRule.onNodeWithText(getStringResource(R.string.new_wallet_1_button)).also {
             it.performClick()
         }
 
-        assertEquals(BackupStage.EducationRecoveryPhrase, testSetup.getStage())
+        assertEquals(BackupStage.EducationRecoveryPhrase.order, testSetup.getStage().order)
```

Comment:
Does the .equals() comparison not end up working for these?  It seems like that would be more robust and would avoid exposing the `order` value.

Reply:
Yes, it works at the end. Thanks.